02Lesson/01Project/index.html

```diff
@@ -1,9 +1,27 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-</head>
-<body>
-
-</body>
+  <head>
+    <title>Party Time</title>
+    <meta charset="utf-8"/>
+  </head>
+  <body>
+    <h1>
+      Party Time
+    </h1>
+    <h3>
+      What we are
+    </h3>
+    <img src="https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcR9XbIxjBhgiqPaWqRskZ3g7bFDCPW4ZEGln44--OShjJ7Brd1Y" alt="Party Time">
+    <p>
+      We are a great team to call for parties who want amazing entertainment and fun
+    </p>
+    <hr>
+    <h3>What we offer</h3>
```

Comment:
I would always recommend to close all block elements on the same level they are opened:
```html
<h3>
   What we offer
</h3>
´´´